akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala

```diff
@@ -143,6 +145,7 @@ object TypedBenchmarkActors {
     }
   }
 
+  @silent("deprecated")
```

Comment:
what is deprecated here?

Reply:
@chbatey `val` keyword in for comprehension
```
/akka/akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala:149:19: 
val keyword in for comprehension is deprecated
[error]       val message = Message(pong) // just allocate once
```

Reply:
Can we fix it then? Just remove the val?